tests/test_api.py

```diff
@@ -1,6 +1,7 @@
 import pytest
 from app import app
 from app.config import Config
+import json
```

Comment:
Why import json here? It is not used.

Reply:
Oops, forgot to remove it indeed. My bad. Fixed in 532228f.